test/language/expressions/postfix-decrement/this.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2023 Veera Sivarajan Ecma International.  All rights reserved.
```

Comment:
The copyright line should indicate the author (or the company who owns the copyright to the author's code)
```suggestion
// Copyright (c) 2023 Veera Sivarajan.  All rights reserved.
```